javascripts/discourse/components/tag-game-card-players-count.gjs

```diff
@@ -0,0 +1,36 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import { bind } from "discourse-common/utils/decorators";
+import dIcon from "discourse-common/helpers/d-icon";
+
+export default class TagGameCardPlayersCount extends Component {
+  @tracked allPlayers;
+  constructor() {
+    super(...arguments);
+    this.allPlayers = this.allPlayerCounts;
+  }
+
+  get allPlayerCounts() {
+    return Array.from(
+      new Set([...this.args.best_players, ...this.args.recommended_players])
+    ).sort((a, b) => parseInt(a) - parseInt(b));
+  }
+
+  @bind
+  getPlayerCountClass(count) {
+    let isBestPlayer = this.args.best_players.includes(count);
+    return (
+      "game-tag-card__player-count game-tag-card__player-count" +
+      (isBestPlayer ? "--best" : "--recommended")
+    );
+  }
+
+  <template>
+    {{#each this.allPlayers as |count|}}
```

Comment:
Doesn't it work if we remove the `allPlayers` tracked attribute and reference `this.allPlayerCounts` directly here instead?

Reply:
Yes, but I had to add the @bind decorator to the Get method. No problem.